Simulation:
    id: ltm_simulator
    control_prefix: /mdb/control
    executed_policy_prefix: /mdb/executed_policy
Sensors:
    -
        id: ball_dist
        ros_name_prefix: /mdb/baxter/sensor/ball_dist
    -
        id: ball_ang
        ros_name_prefix: /mdb/baxter/sensor/ball_ang
    -
        id: ball_size
        ros_name_prefix: /mdb/baxter/sensor/ball_size
    -
        id: box_dist
        ros_name_prefix: /mdb/baxter/sensor/box_dist
    -
        id: box_ang
        ros_name_prefix: /mdb/baxter/sensor/box_ang
    -
        id: box_size
        ros_name_prefix: /mdb/baxter/sensor/box_size
    -
        id: ball_in_left_hand
        ros_name_prefix: /mdb/baxter/sensor/ball_in_left_hand
    -
        id: ball_in_right_hand
        ros_name_prefix: /mdb/baxter/sensor/ball_in_right_hand
    -
        id: ball_in_box
        ros_name_prefix: /mdb/baxter/sensor/ball_in_box
    -
        id: ball_with_robot
        ros_name_prefix: /mdb/baxter/sensor/ball_with_robot
